tests/epyccel/recognised_functions/test_numpy_operators.py

```diff
@@ -0,0 +1,204 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+import os
+from typing import TypeVar
+from numpy.random import randint
+import numpy as np
+import pytest
```

Comment:
I am surprised that Pylint does not complain about not separating the standard library imports from the third-party ones... 🤷‍♂️